lib/tritonapi.js

```diff
@@ -2667,6 +2663,81 @@ TritonApi.prototype.getVolume = function getVolume(name, cb) {
     }
 };
 
+/**
+ * A function appropriate for `vasync.pipeline` funcs that takes a `arg.id`
+ * volume name, shortid or uuid, and determines the volume id (setting it
+ * as `arg.volId`).
+ */
+function _stepVolId(arg, next) {
```

Comment:
nit: Can we move this _stepVolId function up to the section where the other "_step*" functions are defined?